content/addons.md

```diff
@@ -68,27 +68,35 @@ ember addon <addon-name> [options]
+### Addon file structure
 
-```bash
-ember generate in-repo-addon <addon-name> [options]
-```
+In some ways, an addon is like a mini Ember app. It has a very similar file structure, uses a lot of the same API methods, and can do most things that components are able to do. 
 
-This generates a folder called `lib/<addon-name>` that contains its own `package.json` and an `index.js` file. 
--->
+Let's take a look a some of the most important files and folders in an addon, and how they are different from what we would find in an app. 
 
-### Addon file structure
+#### `addon/` 
 
-In some ways, an addon is like a mini Ember app. It has a very similar file structure, uses a lot of the same API methods, and can do most things that components are able to do. 
+If an addon only offers one UI template, the addon files usually go in the `addon/` directory. When the addon is used in a template, it is called by the "real name" of the addon, like `{{my-addon-name someValue=true}}`. This directory can hold many of the same subdirectories and files that an Ember app would, like `/components/` and `/templates/`.
+
+#### `app/` 
+
+If an addon will export multiple templates, the addon's files are referenced from the `app/` directory. When the addon templates are used, they are called by the template name, not the addon name. For example, the addon could provide `{{super-hero-img}}` and `{{fantastic-form}}`. Many addon developers prefix these kinds of templates with a nickname in order to avoid namespace clashes. For example, an addon template named `{{table-row}}` might cause problems if the app using it has a component by the same name, so `{{bs-table-row}}` is safer.
+
+#### `tests/dummy/`
+This directory contains a full Ember app for addon testing purposes. During tests, we can check to make sure that the addon works or looks as expected when it is used in an app. Many addon developers use the dummy app to hold their documentation site's content as well.
+
+#### `package.json` 
+
+If we want other people to be able to use our addon, at minimum, we need to specify a license, a version, the repository url, and description.
+
+#### `config/ember-try.js`
+
+This is a place to configure which versions of Ember that the test suite should check for compatibility.
 
-<!-- add a file tree table and explanations -->
-<!-- include difference between app and addon namespace folders -->
 
-### Creating an addon component template
+## Creating a reusable UI component addon
 
-To create a component template that can be shared between apps, the process is a lot like creating a normal app component:
+To create a UI component template that can be shared between apps, the process is a lot like creating a normal app component:
```

Comment:
To create a UI component template that can be shared between apps, the process is a lot like creating a normal app component:
->
If we want to create a UI component template that can be shared between apps, first we need to create an addon:
```
ember addon <addon-name> 
```
Then  the process is a lot like creating a normal app component: